sync/awsdatabricks.py

```diff
@@ -1017,71 +1047,205 @@ def _s3_contents_have_all_rollover_logs(contents: List[dict], run_end_time_secon
     )
 
 
+def _dbfs_directory_has_all_rollover_logs(contents: dict, run_end_time_millis: float):
+    files = contents["files"]
+    final_rollover_log = contents and next(
+        (
+            file
+            for file in files
+            if Path(file["path"]).stem in {"eventlog", "eventlog.gz", "eventlog.json.gz"}
+        ),
+        False,
+    )
+    return (
+        # Related to - https://docs.databricks.com/clusters/configure.html#cluster-log-delivery-1
+        # We want to make sure that the final_rollover_log has had data written to it AFTER the run has actually
+        # completed. We use this as a signal that Databricks has flushed all event log data to DBFS, and that we
+        # can proceed with the upload + prediction.
+        final_rollover_log
+        and final_rollover_log["modification_time"] >= run_end_time_millis
+    )
```

Comment:
As with S3, check that the most recent event log file has a modification time _after_ our run complete.